clients/lsp-eslint.el

```diff
@@ -233,8 +233,10 @@ source.fixAll code action."
   (lsp-stdio-connection
    (lambda () lsp-eslint-server-command)
    (lambda ()
-     (and (cl-second lsp-eslint-server-command)
-          (file-exists-p (cl-second lsp-eslint-server-command)))))
+     (or
+      (not (equal (f-base (f-filename (cl-first lsp-eslint-server-command))) "node")) ;; if first command is not node then assume it's good
```

Comment:
This line is too long, you should split the comment into different line

Reply:
done!